README.md

```diff
@@ -1,90 +1,89 @@
 # gatsby-remark-mermaid
 [![npm](https://img.shields.io/npm/v/gatsby-remark-mermaid.svg?style=flat-square)](https://www.npmjs.com/package/gatsby-remark-mermaid)
 
-Create [mermaid](https://mermaidjs.github.io/) graphs and diagrams in your markdown files.
+Creates [mermaid](https://mermaidjs.github.io/) graphs and diagrams in your markdown files.
 
-This plugin uses **server-side rendering**. This means the svg is rendered on build time instead of having a runtime
-dependency on mermaid.
+This plugin uses [remark-mermaidjs](https://github.com/remcohaszing/remark-mermaidjs) to generate SVG diagrams at build-time. The mermaid code blocks are replaced with an inline SVGs in the generated HTML. This prevents any runtime
+dependencies on `mermaid.js`.
 
 ## Install
 
-`npm install --save gatsby-remark-mermaid gatsby-transformer-remark puppeteer`
+```bash
+npm install gatsby-remark-mermaid gatsby-transformer-remark
+```
+
+## How to use
```

Comment:
Thank you for structuring the readme!

I’ve been trying to use a more consistent readme layout across my repositories, including consistent headers.

```suggestion
## Usage
```